src/current/releases/index.md

```diff
@@ -116,6 +116,8 @@ The following binaries are not suitable for production environments:
             <td>{{ r.release_date }}</td> {% comment %} Release date of the release. {% endcomment %}
                 {% if r.withdrawn == true %} {% comment %} Suppress withdrawn releases. {% endcomment %}
             <td><span class="badge badge-gray">Withdrawn</span></td>
+                {% elsif r.cloud_only == true %} {% comment %} Suppress download links for Cloud-first releases {% endcomment %}
+            <td><span>Available for select CockroachDB Dedicated customers only</span></td>
```

Comment:
Since they're even using different dates to roll out Dedicated vs. Serverless stuff now, suggest making this text more generic, something like 

"Available for select Cockroach Cloud customers only"

Reply:
I updated this PR to also add fields for a short and long message if you set `cloud_only` to `true` to address this concern. See the example messages in `releases.yml`.